### ISSUE TITLE
pixl-wise distillation loss 

### ISSUE THREAD

**Issue by luhc15** [irfanICMLL/structure_knowledge_distillation#11]
Thanks for your excellent work. 
 I wonder in CriterionPixelWise() funciton:
     N,C,W,H = preds_T[0].shape
     softmax_pred_T = F.softmax(preds_T[0].view(-1,C), dim=1)
Should it be 
      softmax_pred_T = F.softmax(preds_T[0].permute(0,2,3,1).view(-1,C), dim=1)?
Thanks

**Comment by irfanICMLL** [comment]
Thank you for pointing out the mistakes, and we have updated the code~
It should be 'softmax_pred_T = F.softmax(preds_T[0].permute(0,2,3,1).view(-1,C), dim=1)' theoretically, and actually in the old conference version, it does not have this bug.

We will fix this bug and report new results recently.
